### ISSUE TITLE
Manifest File Error - Tried to find Solution all over the Web but no success.

### ISSUE THREAD

**Issue by ShajeeFareedi** [noitcudni/google-webmaster-tools-bulk-url-removal#35]
Failed to load extension
File
E:\Software\Google Webmaster Tools Bulk URLs\google-webmaster-tools-bulk-url-removal-master\google-webmaster-tools-bulk-url-removal-master
Error
Manifest file is missing or unreadable
Could not load manifest.

**Comment by noitcudni** [comment]
Did you compile the code? Did you follow the instructions in the README?

**Comment by ShajeeFareedi** [comment]
Hello, 

Yes, I tried to follow the instructions but not luck. 

First, I downloaded the JAVA and installed
Secondly, I tried to install leiningen, I get Errors.
 - Java Kit could not be found automatically, Ensure that you have downloaded and installed a JDK from: oracle (website url). Well I Clicked Yes.

- After clicking next > Next > Select the path to a Java Development Kit for Leiningen to use:
*A radial box with the text: Custom Location, When I click Next, an pop-up error appeared: Run Time Error (28:617) Out of Range. 

Thats It. How to resolve any of these issues?
